.github/workflows/deploy-production.yml

```diff
@@ -19,7 +18,7 @@ jobs:
     # Map a step output to a job output
     outputs:
       api: ${{ steps.changes.outputs.api }}
-      lambdas: ${{ steps.changes.outputs.lambdas }}
+      infra-lambdas: ${{ steps.changes.outputs.infra-lambdas }}
```

Comment:
should this be `infra-api-lambdas` & `infra-api-lambdas-sandbox`?

Reply:
`api` has its own output